Formula/easy-rsa.rb

```diff
@@ -0,0 +1,54 @@
+class EasyRsa < Formula
+  desc "CLI utility to build and manage a PKI CA"
+  homepage "https://github.com/OpenVPN/easy-rsa"
+  url "https://github.com/OpenVPN/easy-rsa/archive/v3.0.8.tar.gz"
+  sha256 "fd6b67d867c3b8afd53efa2ca015477f6658a02323e1799432083472ac0dd200"
+  license "GPL-2.0-only"
+  head "https://github.com/OpenVPN/easy-rsa.git"
+
+  depends_on "openssl@1.1"
+
+  def install
+    # Rename and edit vars.example file to define easy-rsa variables for homebrew
+    mv "easyrsa3/vars.example", "easyrsa3/vars"
+    inreplace "easyrsa3/vars" do |s|
+      s.gsub! "#set_var EASYRSA\t\"${0%/*}\"",
+              "set_var EASYRSA\t\"#{etc}/pki\""
+      s.gsub! "#set_var EASYRSA_OPENSSL\t\"openssl\"",
+              "set_var EASYRSA_OPENSSL\t\"#{Formula["openssl@1.1"].bin}/openssl\""
+      s.gsub! "#set_var EASYRSA_PKI\t\t\"$PWD/pki\"",
+              "set_var EASYRSA_PKI\t\"$EASYRSA\""
+    end
+
+    bin.install "easyrsa3/easyrsa"
+
+    (etc/"pki").install %w[
+      easyrsa3/openssl-easyrsa.cnf
+      easyrsa3/vars
+      easyrsa3/x509-types
+    ]
+
+    doc.install %w[
+      ChangeLog
+      COPYING.md
+      KNOWN_ISSUES
+      README.md
+      README.quickstart.md
+    ]
+
+    doc.install Dir["doc/*"]
```

Comment:
Is there nothing like a `make install`?

Reply:
No, but it's apparently at the top of their [wishlist](https://github.com/OpenVPN/easy-rsa/blob/master/doc/TODO):

```
Easy-RSA 3 TODO / wishlist

Feature support:
    * makefile
        * It may be useful to port the Makefile from the 2.x series
        * This may be desirable for building and downstream packaging
```

It's all a bit of a hot mess. By default, the `easyrsa` binary and the configuration files are lumped in together, and it's looking for the config files next to the binary and will create a `pki` directory under the current one for the output.

```
$ brew install --interactive easy-rsa
==> Downloading https://github.com/OpenVPN/easy-rsa/archive/v3.0.8.tar.gz
Already downloaded: /Users/chris/Library/Caches/Homebrew/downloads/9721939a50a850a6861cbb7f16953662d77aa181623cd96ebf85d5073ebf2d0e--easy-rsa-3.0.8.tar.gz
==> Entering interactive mode
Type `exit` to return and finalize the installation.
Install to this prefix: /usr/local/Cellar/easy-rsa/3.0.8
$ tree
.
├── COPYING.md
├── ChangeLog
├── KNOWN_ISSUES
├── Licensing
│   ├── gpl-2.0.txt
│   └── mktemp.txt
├── README.md
├── README.quickstart.md
├── build
│   ├── Building.md
│   └── build-dist.sh
├── distro
│   ├── README
│   └── windows
│       ...
├── doc
│   ├── EasyRSA-Advanced.md
│   ├── EasyRSA-Readme.md
│   ├── EasyRSA-Upgrade-Notes.md
│   ├── Hacking.md
│   ├── Intro-To-PKI.md
│   └── TODO
├── easyrsa3
│   ├── easyrsa
│   ├── openssl-easyrsa.cnf
│   ├── vars.example
│   └── x509-types
│       ├── COMMON
│       ├── ca
│       ├── client
│       ├── code-signing
│       ├── email
│       ├── kdc
│       ├── server
│       └── serverClient
├── op_test.orig
├── op_test.sh
├── release-keys
│   └── README.md
├── wop_test.bat
└── wop_test.sh
```

All that's necessary for operation is under the `easyrsa3` directory, plus docs in `.` and `docs`. Everything else is Windows-related. The build script just recreates the tarball from source. (A self-licking ice cream cone?)

I can try to see if I can locate and discern from the 2.x makefile where they might have previously been installing, or we can just place the files from `easyrsa3` wherever is most-consistent with Homebrew / standard practice. I saw that `#{etc}/pki` was suggested for the config files the last time this was proposed (which is why I wrote it that way initially), but as I mentioned elsewhere, it seems like the program nukes that directory, so maybe `#{share}` for those?

Reply:
Here are the latest [`makefile.am`](https://github.com/OpenVPN/easy-rsa/blob/release/2.x/Makefile.am) and [`configure.ac`](https://github.com/OpenVPN/easy-rsa/blob/release/2.x/configure.ac) from their 2.x branch. I've got no experience with `make`, but if I'm interpreting these correctly (and referencing the [README](https://github.com/OpenVPN/easy-rsa/blob/release/2.x/doc/README-2.0)), their `make install` is just lumping everything together in `easy-rsa/` under the current working directory or as specified as well, so I think it's going to be up to us to decide where we want `brew` to place these things by default.

Reply:
I changed the config path to `#{share}` in 05c4940 since the files definitely need to go somewhere other than `#{etc}/pki`. I can move them elsewhere if it would be more appropriate.